v3-docs/docs/cli/using-core-types.md

```
@@ -1,39 +1,177 @@
-Using core types with zodern:types
+## How It Works
 
-For MeteorJS in its version 2.8.1 we have introduced to our core packages an integration with the [zodern:types](https://github.com/zodern/meteor-types) package. 
-This package allows you to use the TypeScript types for the Meteor core packages in your TypeScript code or JavaScript code. 
-in order to use the types you need to install the package by running the command:
+During every build, Meteor scans the compiled packages in your project and
+writes type declaration files to `.meteor/local/types/`:
 
-```bash
-meteor add zodern:types
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)